app/Http/Controllers/Chat/ChannelsController.php

```diff
@@ -243,6 +244,10 @@ public function store()
             if ($channel->exists) {
                 $channel->addUser($sender);
             }
+        } else if ($params['type'] === Channel::TYPES['multiplayer']) {
+            $room = Room::findOrFail($params['target_id']);
+            $channel = $room->channel;
+            $channel->addUser($sender);
```

Comment:
I'm not sure this is necessary since `Room` includes the `channel_id` so you can use the other endpoint instead 🤔